test/cppsuite/test_harness/workload/database_model.h

```diff
@@ -56,18 +56,37 @@ struct collection_t {
 /* Representation of the collections in memory. */
 class database {
     public:
-    const std::vector<std::string>
-    get_collection_names() const
+    std::mutex &
+    get_mtx()
```

Comment:
Can you add the "const" keyword after the function declaration ? Would it compile ?

Reply:
Don't think so. We're returning a non-const ref to a member. We need to call `std::mutex::lock` which is not `const` so it needs to be `std::mutex &` and not `const std::mutex &`.

Reply:
I meant `std::mutex& get_mtx() const`